atomics/include/desul/atomics/Lock_Array_HIP.hpp

```diff
@@ -120,13 +114,8 @@ __device__ inline void unlock_address_hip(void* ptr, desul::MemoryScopeNode) {
   offset = offset & HIP_SPACE_ATOMIC_MASK;
   atomicExch(&desul::Impl::HIP_SPACE_ATOMIC_LOCKS_NODE[offset], 0);
 }
-#endif
```

Comment:
If you look close enough you will notice that the open/close namespaces were unbalanced